StripePaymentSheet/StripePaymentSheet/Source/PaymentSheet/Saved Payment Method Screen/SavedPaymentOptionsViewController.swift

```diff
@@ -71,13 +71,24 @@ class SavedPaymentOptionsViewController: UIViewController {
         let removeSavedPaymentMethodMessage: String?
         let merchantDisplayName: String
         let isTestMode: Bool
+        let allowRemovalOfLastSavedPaymentMethod: Bool
     }
 
-    var hasRemovablePaymentMethods: Bool {
-        return (
-            configuration.customerID != nil &&
-            !savedPaymentMethods.isEmpty
-        )
+    // MARK: - Internal Properties
+
+    /// Whether or not you can edit save payment methods by removing or updating them.
+    var canEditPaymentMethods: Bool {
+        switch savedPaymentMethods.count {
+        case 0:
+            return false
+        case 1:
+            // If there's exactly one PM, customer can only edit if configuration allows removal or if that single PM is editable
```

Comment:
maybe replace "is editable"
with "allows for the card brand choice to be updated."